01_fib/harirakr/main_test.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNegativeFib(t *testing.T) {
+
```

Comment:
Unnecessary blank line.
No need for blank lines at beginning or end of block.
Use blank lines between type, method and function definitions and within a block for logical separation.

Belowaswell.

Reply:
Done